helm/csi-isilon/values.yaml

```diff
@@ -387,8 +387,6 @@ podmon:
 
 encryption:
   # enable: Enable/disable volume encryption feature
-  # Encryption cannot be enabled in tandem with resizer or authorization.
-  # Enable this feature only after contacting support for additional information.
   enabled: false
```

Comment:
On the previous line(#389), shouldn't it be "enabled:" instead of "enable:" ?

Reply:
Thanks for spotting that. Fixed.